front/pages/.well-known/security.txt.ts

```diff
@@ -0,0 +1,19 @@
+import type { GetServerSideProps } from "next";
+
+export const getServerSideProps: GetServerSideProps = async ({ res }) => {
```

Comment:
@zmarouf You can make it easier by just putting the raw txt file directly here: `front/public/.well-known/security.txt` 🙇🏻‍♀️